src/confluent_kafka/schema_registry/json_schema.py

```diff
@@ -43,6 +43,36 @@ def __exit__(self, *args):
         return False
 
 
+def _id_of(schema):
+    """
+    Returns the schema id if present otherwise None.
+    :param schema: Schema to return id of.
+    :return: Id of schema if present otherwise None.
+    """
+    return schema.get('$id', "None")
```

Comment:
shouldn't be `None` here, without quotes?

Reply:
Thx, I've updated this.

Reply:
Makes sense. Updated.

Reply:
If "id" field is mandatory, shouldn't we raise the error?

Reply:
It's not a mandatory field, but is used as a best practice and is needed if you need to reference other schemas.

Reply:
https://json-schema.org/understanding-json-schema/structuring.html#id

Reply:
Probably for our use case it's mandatory, we're using it as the key in a map, it could throw a ValueError in that case

Reply:
Yeah, so should we throw an exception here?

Reply:
Thanks. I've updated the code to use ref.name as the id.